lib/senkyoshi/models/forum.rb

```diff
@@ -22,6 +32,18 @@ def canvas_conversion(course, _resources = nil)
       discussion.identifier = @id
       discussion.discussion_type = @discussion_type
       course.discussions << discussion
+      if @module_item
+        master_module = course.canvas_modules.
```

Comment:
I'm seeing this same detect repeated. Consider making a method in the course to dry this up? Perhaps make a instance variable to hold the `canvas_modules` in course.rb?